pkg/deployer/clusterapiservertemplate.go

```diff
@@ -126,7 +126,7 @@ kind: ServiceAccount
 apiVersion: v1
 metadata:
   name: apiserver
-  namespace: default
+  namespace: {{ .Namespace }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
```

Comment:
To be checked, but I believe name of this ClusterRoleBinding should be `{{ .Namespace }}:system:auth-delegator` instead of `default:system:auth-delegator`.

Reply:
I think so too. I'll confirm